recipes/libnvjpeg/meta.yaml

```diff
@@ -0,0 +1,145 @@
+        - test -L $PREFIX/lib/libnvjpeg.so.{{ version.split(".")[0] }}                            # [linux]
+        - test -L $PREFIX/targets/{{ target_name }}/lib/libnvjpeg.so.{{ version }}                # [linux]
+        - test -f $PREFIX/targets/{{ target_name }}/lib/libnvjpeg.so.{{ version.split(".")[0] }}  # [linux]
+        - if not exist %LIBRARY_BIN%\nvjpeg64_{{ version.split(".")[0] }}.dll exit 1              # [win]
+    about:
+      home: https://developer.nvidia.com/cuda-toolkit
+      license_file: LICENSE
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_url: https://docs.nvidia.com/cuda/eula/index.html
+      summary: NVJPEG native runtime libraries
+      description: |
+        NVJPEG native runtime libraries
+      doc_url: https://docs.nvidia.com/cuda/index.html
+
+  - name: libnvjpeg-dev
+    build:
+      run_exports:
+        - {{ pin_subpackage("libnvjpeg", max_pin="x") }}
+    files:
+      - lib/libnvjpeg.so                                  # [linux]
+      - lib/pkgconfig/nvjpeg*.pc                          # [linux]
+      - targets/{{ target_name }}/lib/libnvjpeg.so        # [linux]
+      - targets/{{ target_name }}/lib/stubs/libnvjpeg.so  # [linux]
+      - targets/{{ target_name }}/include/nvjpeg*         # [linux]
+      - Library\include                                   # [win]
+      - Library\lib\nvjpeg.lib                            # [win]
+    requirements:
+      build:
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+      host:
+        - cuda-version {{ cuda_version }}
+      run:
+        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+        - {{ pin_subpackage("libnvjpeg", exact=True) }}
+      run_constrained:
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - libnvjpeg-static >={{ version }}      # [linux]
+    test:
+      commands:
+        - test -L $PREFIX/lib/libnvjpeg.so                                    # [linux]
+        - test -f $PREFIX/lib/pkgconfig/nvjpeg*.pc                            # [linux]
+        - test -f $PREFIX/targets/{{ target_name }}/include/nvjpeg.h          # [linux]
+        - test -L $PREFIX/targets/{{ target_name }}/lib/libnvjpeg.so          # [linux]
+        - test -f $PREFIX/targets/{{ target_name }}/lib/stubs/libnvjpeg.so    # [linux]
+        - if not exist %LIBRARY_INC%\nvjpeg.h exit 1                          # [win]
+        - if not exist %LIBRARY_LIB%\nvjpeg.lib exit 1                        # [win]
+    about:
+      home: https://developer.nvidia.com/cuda-toolkit
+      license_file: LICENSE
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_url: https://docs.nvidia.com/cuda/eula/index.html
+      summary: NVJPEG native runtime libraries
+      description: |
+        NVJPEG native runtime libraries
+      doc_url: https://docs.nvidia.com/cuda/index.html
```

Comment:
Consider using https://docs.nvidia.com/cuda/nvjpeg/index.html or https://developer.nvidia.com/nvjpeg for the `doc_url`.